aten/src/ATen/ConjugateFallback.cpp

```diff
@@ -0,0 +1,122 @@
+#include <ATen/ATen.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <torch/library.h>
+
+#include <ATen/native/UnaryOps.h>
+#include <ATen/NativeFunctions.h>
+
+namespace at {
+
+void conjugateFallback(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
+  // Situations to handle:
+  //  1. Purely functional situation.  Easy: materialize all inputs and
+  //     call it a day.
+  //  2. Inplace operation.  Desugar x.add_(2) into x.conj_().add_(2).conj_().
+  //     Materialize other inputs as in (1).
+  //  3. Out-of-place operation.  Desugar add(x, 2, out=y) into y.copy_(add(x, 2))
+  //     Materialize other inputs as in (1).
+  //
+  // It is important to be able to tell if we READ from an argument and if we
+  // WRITE from an argument.  Conservative approach is to assume that we always
+  // READ from an argument, but in out-of-place operations you can skip
+  // conjugating inputs on entry that never get used.  In current schema we
+  // can't easily tell if inplace situation has happened, so don't do it.
+
+  // std::cerr << "conj fallback " << op.schema().name() << "\n";
+
+  const auto& arguments = op.schema().arguments();
+  const auto num_arguments = arguments.size();
+  const auto stack_start = stack->size() - num_arguments;
+
+  c10::optional<bool> is_write;
+  for (int64_t i = 0; i < num_arguments; ++i) {
+    const auto& alias_info = arguments[i].alias_info();
```

Comment:
alias_info contains info about whether the tensor will get mutated in the operation